tests/test_integration.py

```diff
@@ -132,6 +132,7 @@ def setUp(self):
             self.vm_obj['aws_key_name'] = 'eu-central-1-key'
             self.vm_obj['service_groups'] = [
                 'igvm-integration.test.sg',
+                'ig-aws-base-internet.sg',
```

Comment:
We maybe should add a TODO here to get rid of this group.

Reply:
Done.